example/provider-local/managedseedset/managedseedset.yaml

```
@@ -0,0 +1,72 @@
+# Please note: The `ManagedSeedSet` is a WIP feature. This example is primarily meant to support further development in the local setup.
+apiVersion: seedmanagement.gardener.cloud/v1alpha1
+kind: ManagedSeedSet
+metadata:
+  name: my-mss
+  namespace: garden # Must be garden
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: my-managed-seed-set
+  template:
+    # <See `example/55-managedseed-gardenlet.yaml` for more details>
+    metadata:
+      labels:
+        name: my-managed-seed-set
+    spec:
+      # shoot.name must not be specified
+      gardenlet:
+        config: # GardenletConfiguration resource
+          # <See `example/20-componentconfig-gardenlet.yaml` for more details>
+          apiVersion: gardenlet.config.gardener.cloud/v1alpha1
+          kind: GardenletConfiguration
+          seedConfig:
+            metadata:
+              labels:
+                name: my-managed-seed-set
+            spec:
+              dns:
+                ingress:
+                  # "replica-name" in DNS domains will be replaced by the actual replica name
+                  domain: replica-name.ingress.dev.my-seed.example.com
```

Local setup manifest, we should use a domain related to the local setup:

```suggestion
                  domain: replica-name.ingress.local.seed.local.gardener.cloud
```

---

Done via: [`ef3f739` (#13572)](https://github.com/gardener/gardener/pull/13572/commits/ef3f739914975c5a3d652b5b0a2b2bbcca610897)